pyproject.toml

```diff
@@ -165,4 +166,5 @@ fixable = [
     "B",  # bugbear
     "I",  # isort
     "RUF", # ruff
+    "UP", # pyupgrade
```

Comment:
Doesn't `ruff` have these as `fixable` as well? At work we don't put these in our config and still works.

Reply:
I checked, you're right ! [ec40ff0](https://github.com/pylint-dev/pylint/pull/9086/commits/ec40ff076261bd1ba4e86fa6031c90018615c207)